### PR TITLE
RMP-228 Standardised oxd icon sizes and selectinputbutton.vue content alignment fix

### PR DESCRIPTION
**Pull request by gayan-orangehrm**
This PR includes,

1. Introduced sizes for icons discussing with Mike and standardised sizes throughout the application.
2. Spread contents of SelectInputButton.vue using justify space between

### CODE REVIEW

File: components/src/core/components/Input/Select/select-input.scss [hunk 1]
```diff
@@ -196,20 +196,23 @@
 .dropdown-btn {
   display: flex;
   align-items: center;
-  justify-content: space-between;
+  justify-content: space-between !important;
```
Comment (lgtkaushalya) [inline components/src/core/components/Input/Select/select-input.scss:199]: Remove this generic change and handle only the scenario for this page. Pass it as a style.

Comment (gayan-orangehrm) [reply components/src/core/components/Input/Select/select-input.scss:199]: removed and applied from candidates list configurations as an inline css

File: components/src/core/components/Icon/icon.scss [hunk 2]
```diff
@@ -5,16 +5,49 @@
...
+  }
 }
 
 .oxd-svg-icon {
   display: flex;
-  height: 1rem;
-  width: 1rem;
   fill: currentColor;
+  &--extra-small {
+    width: $oxd-svg-icon-font-size--extra-small !important;
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Icon/icon.scss:32]: Removed it

File: components/src/core/components/Button/Button.vue [hunk 3]
```diff
@@ -7,9 +7,9 @@
     :flow="flow"
   >
     <slot name="icon">
-      <oxd-icon v-if="iconName" :name="iconName" class="oxd-button-icon" />
+      <oxd-icon v-if="iconName" :name="iconName" :size="iconSize" :style="iconStyle" class="oxd-button-icon" />
     </slot>
-    <div class="oxd-button-label-wrapper">
+    <div class="oxd-button-label-wrapper mx-w-100">
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Button/Button.vue:12]: Removed

**Pull request merged**